<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSettingDialogForm</class>
 <widget class="QDialog" name="EditSettingDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Settings on Sequence Editing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QRadioButton" name="resizeRadioButton">
     <property name="text">
      <string>Expand or crop affected annotation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="removeRadioButton">
     <property name="text">
      <string>Remove affected annotation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="splitRadioButton">
     <property name="text">
      <string>Split (join annotation parts)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="split_separateRadioButton">
     <property name="text">
      <string>Split (separate annotations parts)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="topMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QCheckBox" name="recalculateQuals">
       <property name="text">
        <string>Recalculate values of qualifiers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSettingDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSettingDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
